package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "app",
+  "private": "true",
+  "scripts": {
+    "build": "webpack --config webpack.config.js",
+    "fix-styles": "npx stylelint '**/*.{css,scss}' --fix"
+  },
+  "dependencies": {
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "@babel/preset-react": "^7.22.15",
+    "@reduxjs/toolkit": "^1.9.7",
+    "axios": "^1.5.1",
+    "babel-loader": "^9.1.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.1.3",
+    "react-router-dom": "^6.17.0"
+  },
+  "devDependencies": {
+    "install": "^0.13.0",
+    "npm": "^10.2.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^5.1.4"
+  }
```

Comment:
- [ ] While I'm precompiling the assets to run the app there is an issue raised because `webpack` package is not installed. please install it `npm install webpack` to fix that issue.

![image](https://github.com/FaranoshAmini/hello_rails_react/assets/68796232/ffae8fdc-17b0-44b7-8292-5ae5fed0b895)